ironfish-zkp/src/util.rs

```diff
@@ -45,3 +45,47 @@ pub fn commitment_full_point(
     // Compute final commitment
     (NOTE_COMMITMENT_RANDOMNESS_GENERATOR * rcm) + hash_of_contents
 }
+
+/// Computes the note commitment with sender address, returning the full point.
+pub fn commitment_full_point_new(
+    asset_generator: jubjub::SubgroupPoint,
+    value: u64,
+    pk_d: jubjub::SubgroupPoint,
```

Comment:
lets rename this to `owner_address` to be consistent with below.

Also lets take the opportunity to rename `rcm` to `randomness` or `commitment_randomness`

Reply:
I can rename rcm, but I am not sure about pk_d. That pk_d is the name used in zkp. In our use case, pk_d is equal to the owner address because we removed the diversifier. But in general, pk_d is part of a public address, more details here https://www.figma.com/file/fqTb2xt5fgGTP9MEeJQPmu/Sapling-Keys?node-id=0%3A1&t=1ErVhzYWZUOSonu2-0